packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -713,6 +712,7 @@ class APIProviderFixture {
    */
   constructor({ mocks = {} } = {}) {
     this._pages = [];
+    this._termAutoIncrementId = 0;
```

Comment:
should start with `id = 1`. Wordpress reserves the id `0` for the 'no parent' case so it could interfere with the category input

Our category input uses 0 as our 'null value'

```suggestion
    this._termAutoIncrementId = 1;
```

Reply:
ACTUALLY it should start at 4 😓 the mocks have terms with ids `[1, 2, 3]` already
```suggestion
    this._termAutoIncrementId = 4;
```

Reply:
oh yup. good call

Reply:
I'm just gonna make it `100` so no matter how many we set for initial load, they always have a unique id